core/src/HIP/Kokkos_HIP_Parallel_Team.hpp

```diff
@@ -165,8 +160,7 @@ class TeamPolicyInternal<Kokkos::Experimental::HIP, Properties...>
         static_cast<size_t>(vector_length()),
         static_cast<size_t>(team_scratch_size(0)) + 2 * sizeof(double),
         static_cast<size_t>(thread_scratch_size(0)) + sizeof(double));
-    // FIXME_HIP See comment about teams larger than 256 above
-    return std::min(block_size / vector_length(), 256);
+    return std::min(block_size / vector_length(), 1024);
```

Comment:
Remove the `std::min`

Reply:
7943a27511ba51cc14cd7a2c172613abb6041c43